content/design-systems-for-developers/react/en/build.md

```diff
@@ -52,6 +52,7 @@ Nice, we’ve set up a component explorer!
 By default, Storybook has created a folder `src/stories` with some example stories. However, when we copied our components over, we brought their stories too. We can get them indexed in our Storybook by changing the search path in `.storybook/config.js` from `’src/stories’` to `’src’`, and removing the `src/stories` directory:
 
 ```javascript
+// config.js
```

Comment:
Is it worth having the full file path aka `.storybook/config.js`? I think that would lead to less confusion. Thoughts?